lib/jsdom/browser/default-stylesheet.js

```diff
@@ -173,36 +160,12 @@ h1 {
     font-weight: bold
 }
 
-:-webkit-any(article,aside,nav,section) h1 {
+article h1, aside h1, nav h1, section h1 {
     font-size: 1.5em;
     -webkit-margin-before: 0.83__qem;
     -webkit-margin-after: 0.83em;
 }
 
-:-webkit-any(article,aside,nav,section) :-webkit-any(article,aside,nav,section) h1 {
```

Comment:
Will nwmatcher work with `:matches()` instead of `:-webkit-any()`?

If not, we should add a comment here explaining that we are going against spec for now.